ui/main/src/app/app.component.spec.ts

```diff
@@ -1,178 +0,0 @@
-/* Copyright (c) 2020, RTE (http://www.rte-france.com)
```

Comment:
Why delete all tests for app component (even those unrelated to feature)? Were they too brittle ?

Reply:
To remove button ,i needed de refactor the way authentication is done and refactor the app.component . After that refactor ,unit testing of this component is not needed